tests/system/Models/DeleteModelTest.php

```diff
@@ -148,8 +148,8 @@ public function testOnlyDeleted(): void
     }
 
     /**
-     * If where condition is set, beyond the value was empty (0,'', NULL, etc.),
-     * Exception should not be thrown because condition was explicity set
+     * If where condition is set, beyond the value was empty (0, '', NULL, etc.),
+     * Exception should not be thrown because condition was explicitly set
```

Comment:
What does "beyond the value was empty" mean?

Reply:
Not sure! It seems to be explicitly allowing falsy values?

Reply:
I added `''` as a test case,  because it is here in the comment.
But Postgres does not accept it.

```
1) CodeIgniter\Models\DeleteModelTest::testDontThrowExceptionWhenSoftDeleteConditionIsSetWithEmptyValue with data set #3 ('')
ErrorException: pg_query(): Query failed: ERROR:  invalid input syntax for type integer: ""
LINE 2: WHERE "id" = ''
                     ^

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Database/Postgre/Connection.php:140
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Database/BaseConnection.php:666
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Database/BaseConnection.php:593
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Database/BaseBuilder.php:2079
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Model.php:412
/home/runner/work/CodeIgniter4/CodeIgniter4/system/BaseModel.php:985
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Models/DeleteModelTest.php:163
phpvfscomposer:///home/runner/work/CodeIgniter4/CodeIgniter4/vendor/phpunit/phpunit/phpunit:97
```

Reply:
Fixed the comment, and removed the failed test case (`''`).